src/components/forms/CollectionForm.tsx

```diff
@@ -58,11 +47,13 @@ export const CollectionForm = ({ collection: loadedCollection }: TProps) => {
     const collectionData: ICollectionFormPost = {
       title: collection.title,
       description: collection.description,
-      organisation: collection.organisation as TOrganisation,
```

Comment:
`organisation` is a required field in the admin backend, so whatever is in the final object to be posted must contain an `organisation` field. You can decode the user token (like we do in the family form) to get the organisation to pass back to the backend until we have a change to remove the vestiage of organisation from the admin-backend DTOs though

Reply:
Have just seen this snippet - so all good my end. Will approve!
https://github.com/climatepolicyradar/navigator-admin-frontend/pull/99/files#r1705679747

Reply:
This only moves the assignment of value to organisation to lower in the code where we decide whether it's an update or create so we can reuse the same object. You can see a bit lower down that I add organisation from the loadedFamily to this object when it's an update. 
A ticket to remove organisation from the backend DTO can be found [here](https://linear.app/climate-policy-radar/issue/PDCT-1304/remove-organisation-field-from-collectiondto)